src/components/genericListMaker.js

```diff
@@ -88,6 +91,26 @@ const genericListMaker = (componentName, FormComponent) => {
                       onClick={() => handleDragShow(componentName)}
                     />
                   ) : null}
+                  {editMode &&
```

Comment:
Optimize this code, rather than calling handleVisibilitybutton function twice, use nested ternary functions.